app/src/main/java/io/appium/uiautomator2/utils/AlertHelpers.java

```diff
@@ -36,12 +36,20 @@
 
 public class AlertHelpers {
     private static final String alertButtonResIdPattern = "^android:id/button\\d+$";
-    private static final String alertContentResId = "android:id/content";
+    private static final String alertTitleResId = "android:id/alertTitle";
```

Comment:
![image](https://user-images.githubusercontent.com/3530074/39545972-89174310-4e5b-11e8-9e35-f282c40cb431.png)

Reply:
yep. We can not correctly detect alerts on Android.

Reply:
haha, that's a great proposal 🤣

Reply:
I didn't expect to detect them all, but for most of them this should work though